tests/Drupal/InfoFiles/DependenciesArrayUnitTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Drupal\Test\InfoFiles;
+
+use Drupal\Test\CoderSniffUnitTest;
+
+/**
+ * A sniff unit test checks a .inc file for expected violations of a single
+ * coding standard. Expected errors and warnings are stored in this class.
+ */
+class DependenciesArrayUnitTest extends CoderSniffUnitTest
+{
+
+
+    /**
+     * Returns the lines where errors should occur.
+     *
+     * The key of the array should represent the line number and the value
+     * should represent the number of errors that should occur on that line.
+     *
+     * @param string $testFile The name of the file being tested.
+     *
+     * @return array<int, int>
+     */
+    protected function getErrorList(string $testFile): array
+    {
+        return [];
+
+    }//end getErrorList()
+
+
+    /**
+     * Returns the lines where warnings should occur.
+     *
+     * The key of the array should represent the line number and the value
+     * should represent the number of warnings that should occur on that line.
+     *
+     * @param string $testFile The name of the file being tested.
+     *
+     * @return array<int, int>
+     */
+    protected function getWarningList(string $testFile): array
+    {
+        return [1 => 3];
```

Comment:
tests are failing because this means "3 warnings on line 1". But your sniff does not produce warnings, it makes errors, right?